com.simplenote.Simplenote.appdata.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.simplenote.Simplenote</id>
+  <launchable type="desktop-id">com.simplenote.Simplenote.desktop</launchable>
+  <name>Simplenote</name>
+  <summary>Simplenote Desktop App</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0</project_license>
+  <developer_name>Automattic Inc.</developer_name>
+  <url type="homepage">https://automattic.com/</url>
+  <description>
+    <p>
+      The simplest way to keep notes
+    </p>
+    <p>
+      NOTE: This wrapper is not verified by, affiliated with, or supported by Automattic Inc.
+    </p>
+  </description>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://raw.githubusercontent.com/flathub/com.simplenote.Simplenote/master/screenshots/simplenote1.png</image>
```

Comment:
The build will be failing as these files do not exist yet. Can you change it to your personal fork for now?

Reply:
They have been changed to my personal fork.